# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../vdr/mp3-gcc41.diff
# Copyright (C) 2006 The T2 SDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./mp3/decoder-ogg.h~	2005-02-06 13:42:26.000000000 +0200
+++ ./mp3/decoder-ogg.h	2006-01-08 19:08:05.000000000 +0200
@@ -49,7 +49,7 @@
   ~cOggFile();
   bool Open(bool log=true);
   void Close(void);
-  long long cOggFile::Seek(long long posMs=0, bool relativ=false);
+  long long Seek(long long posMs=0, bool relativ=false);
   int Stream(short *buffer, int samples);
   bool CanSeek(void) { return canSeek; }
   long long IndexMs(void);
